core-java-modules/core-java-date-operations-4/src/test/java/com/baeldung/datetounixtimestamp/DateToUnixTimeStampUnitTest.java

```diff
@@ -0,0 +1,56 @@
+package com.baeldung.datetounixtimestamp;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.time.Instant;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.GregorianCalendar;
+import java.util.TimeZone;
+
+import org.joda.time.DateTime;
+import org.junit.jupiter.api.Test;
+
+class DateToUnixTimeStampUnitTest {
+
+    @Test
+    void givenDate_whenUsingInstantClass_thenConvertToUnixTimeStamp() {
+        Instant givenDate = Instant.parse("2020-09-08T12:16:40Z");
+
+        assertEquals(1599567400L, givenDate.getEpochSecond());
+    }
+
+    @Test
+    void givenDate_whenUsingDateClass_thenConvertToUnixTimeStamp() throws ParseException {
+        SimpleDateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss");
+        dateFormat.setTimeZone(TimeZone.getTimeZone("UTC"));
+        Date givenDate = dateFormat.parse("2023-10-15 22:00:00");
+
+        assertEquals(1697407200000L, givenDate.getTime());
+    }
+
+    @Test
+    void givenDate_whenUsingCalendarClass_thenConvertToUnixTimeStamp() throws ParseException {
```

Comment:
It seems ParseException is never thrown.

Reply:
removed